aspnetcore/host-and-deploy/web-farm.md

```diff
@@ -0,0 +1,77 @@
+ms.author: riande
+ms.custom: mvc
+ms.date: 07/16/2018
+uid: host-and-deploy/web-farm
+---
+# Host ASP.NET Core in a web farm
+
+By [Luke Latham](https://github.com/guardrex) and [Chris Ross](https://github.com/Tratcher)
+
+A *web farm* is a group of two or more web servers (or *nodes*) that host multiple instances of an app. When requests from users arrive to a web farm, a *load balancer* distributes the requests to the web farm's nodes. Web farms improve:
+
+* Reliability/availability &ndash; When one or more nodes fail, the load balancer can route requests to other functioning nodes to continue processing requests.
+* Capacity/performance &ndash; Multiple nodes can process more requests than a single server. The load balancer balances the workload by distributing requests to the nodes.
+* Scalability &ndash; When more or less capacity is required, the number of active nodes can be increased or decreased to match the workload. Web farm platform technologies, such as [Azure App Service](https://azure.microsoft.com/services/app-service/), can automatically add or remove nodes at the request of the system administrator or automatically without human intervention.
+* Maintainability &ndash; Nodes of a web farm can rely on a set of shared services, which results in easier system management. For example, the nodes of a web farm can rely upon a single database server and a common network location for static resources, such as images and downloadable files.
+
+This topic describes configuration and dependencies for ASP.NET core apps hosted in a web farm that rely upon shared resources.
+
+## General configuration
+
+<xref:host-and-deploy/index>  
+Learn how to set up hosting environments and deploy ASP.NET Core apps. Configure a process manager on each node of the web farm to automate app starts and restarts. Each node requires the ASP.NET Core runtime. For more information, see the topics in the [Host and deploy](xref:host-and-deploy/index) area of the documentation.
+
+<xref:host-and-deploy/proxy-load-balancer>  
+Learn about configuration for apps hosted behind proxy servers and load balancers, which often obscure important request information.
+
+<xref:host-and-deploy/azure-apps/index>  
+[Azure App Service](https://azure.microsoft.com/services/app-service/) is a [Microsoft cloud computing platform service](https://azure.microsoft.com/) for hosting web apps, including ASP.NET Core. App Service is a fully managed platform that provides automatic scaling, load balancing, patching, and continuous deployment.
+
+## App data
+
+When an app is scaled to multiple instances, there might be app state that requires sharing across nodes. If the state is transient, consider sharing the state by sharing an [IDistributedCache](/dotnet/api/microsoft.extensions.caching.distributed.idistributedcache). If the shared state requires persistence, consider storing the shared state in a database.
```

Comment:
Should this 2nd sentence say "by using an" instead of "by sharing an"?

Reply:
"by using an" was kind'a what I was doing when I originally said "by implementing an." @Tratcher said, 'use sharing' there. https://github.com/aspnet/Docs/pull/7602#discussion_r202476800

The repeated word bothers me, too. Synonyms aren't easy to come by on this one. I'll try ...

> If the state is transient, consider sharing an \[IDistributedCache](/dotnet/api/microsoft.extensions.caching.distributed.idistributedcache).